pkg/catalog/catalog.go

```diff
@@ -466,16 +466,6 @@ func isSupportedImage(imgName string) bool {
 		"centos/nodejs-12-centos7:latest",
 		"rhscl/nodejs-10-rhel7:latest",
 		"rhscl/nodejs-12-rhel7:latest",
-		"bucharestgold/centos7-s2i-nodejs:latest",
-		"nodeshift/centos7-s2i-nodejs:latest",
-
-		// older images which we should remove soon
-		"rhoar-nodejs/nodejs-8:latest",
-		"rhoar-nodejs/nodejs-10:latest",
-		"bucharestgold/centos7-s2i-nodejs:8.x",
-		"bucharestgold/centos7-s2i-nodejs:10.x",
```

Comment:
same as https://github.com/openshift/odo/pull/3587/files?short_path=7bd4a92#r455565860

Reply:
@mik-dass This image support has been removed in pr https://github.com/openshift/odo/pull/3414, but unfortunately the reference is left behind in the catalog.go file

Reply:
These are all deprecated images